google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITInstanceAdminTest.java

```diff
@@ -45,11 +45,12 @@
 @Category(SerialIntegrationTest.class)
```

Comment:
As this is now always using an owned instance, you can probably change this to a `ParallelIntegrationTest`.